hazelcast/src/main/java/com/hazelcast/spi/Operation.java

```diff
@@ -571,15 +571,20 @@ public final void readData(ObjectDataInput in) throws IOException {
 
     protected abstract void readInternal(ObjectDataInput in) throws IOException;
 
+    protected void toString(StringBuilder sb) {
+    }
+
```

Comment:
Sure, plz look at my comment beginning with "There's still a slight problem here. " (it's folded now)